### PR TITLE
Use a plugin to maintain the license automatically

### PR DESCRIPTION
**Pull request by tiwanari**
I made a change to use [the same plugin](https://github.com/hierynomus/license-gradle-plugin) that is used in [drive](https://github.com/PileProject/drive) to maintain LICENSE automatically. 

For the purpose, I also changed the `.travis.yml` to run a test and license check (it did nothing).

### NOTE
I changed the name of LICENSE author and the e-mail.

### CODE REVIEW

File: src/main/java/com/pileproject/drivecommand/model/ev3/Ev3Constants.java [hunk 1]
```diff
@@ -1,184 +1,182 @@
...
-    public static final byte L_MOTOR_SPEED = (byte) 0x02;
-    /**
-     * M-Motor
-     */
-    public static final byte M_MOTOR_DEGREE = (byte) 0x00;
-    public static final byte M_MOTOR_ROTATE = (byte) 0x01;
-    public static final byte M_MOTOR_SPEED = (byte) 0x02;
-
-}
+/**
```
Comment (tiwanari) [inline src/main/java/com/pileproject/drivecommand/model/ev3/Ev3Constants.java:1]: This file had CRLF line terminators

File: src/main/java/com/pileproject/drivecommand/model/nxt/InputValues.java [hunk 2]
```diff
@@ -1,55 +1,53 @@
...
-     * The first value will be the raw value, but after that it produces scaled values.
-     * With the touch sensor, off scales to 0 and on scales to 1.
-     */
-    public short scaledValue;
-    /**
-     * Currently unused.
-     */
-    public short calibratedValue;
-}
+/**
```
Comment (tiwanari) [inline src/main/java/com/pileproject/drivecommand/model/nxt/InputValues.java:1]: This file had CRLF line terminators

File: src/main/java/com/pileproject/drivecommand/model/nxt/NxtConstants.java [hunk 3]
```diff
@@ -1,156 +1,154 @@
...
-    public static final byte TRANSITIONCNTMODE = 0x40;
-    public static final byte PERIODCOUNTERMODE = 0x60;
-    public static final byte PCTFULLSCALEMODE = (byte) 0x80;
-    public static final byte CELSIUSMODE = (byte) 0xA0;
-    public static final byte FAHRENHEITMODE = (byte) 0xC0;
-    public static final byte ANGLESTEPSMODE = (byte) 0xE0;
-    public static final byte SLOPEMASK = 0x1F;
-    public static final byte MODEMASK = (byte) 0xE0;
-}
+/**
```
Comment (tiwanari) [inline src/main/java/com/pileproject/drivecommand/model/nxt/NxtConstants.java:1]: This file had CRLF line terminators

File: src/main/java/com/pileproject/drivecommand/model/pile/PileConstants.java [hunk 4]
```diff
@@ -1,81 +1,79 @@
...
-        MotorDir(byte value) {
-            mValue = value;
-        }
-
-        public byte value() {
-            return mValue;
-        }
-    }
-}
+/**
```
Comment (tiwanari) [inline src/main/java/com/pileproject/drivecommand/model/pile/PileConstants.java:1]: This file had CRLF line terminators

**Review by mandaiy — APPROVED** [review]

**Pull request merged**